pkg/build/builder/docker.go

```diff
@@ -413,6 +419,33 @@ func traverseAST(cmd string, node *parser.Node) int {
 	return index
 }
 
+// insertEnvAfterFrom inserts an ENV Dockerfile instruction right after the
+// FROM instruction of the given dockerfile to define the environment variables
```

Comment:
> to define the environment variables from env - this half of the the sentence is a bit confusing. Either make it more clear or nuke it.

I won't nuke it. Any proposal of a better sentence?
Again, *env* refers to the argument name. See for example how godocs in the standard library are written: http://docs.golang.org/src/bytes/bytes.go?s=7726:7905#L319. It is common practice to use the argument names in the documentation.